This message is for when a pip declares an output directory and it, or some other pip, also declares a dependency on a source file that is in that directory.


Example:
```
d:\o2\Build\ProductBuild\dsc\liblet_memoryapi_x64_debug\spec.0.dsc(14715,63): error DX14100: Invalid graph since 'd:\o2\Build\oacrdomino\liblet_memoryapi_x64_debug\oacr-merge', produced by 'PipEA4F8CD3562CB074, ProcessRunner.exe, liblet_memoryapi_x64_debug, n4834187996086921314, Common | {oacr merge for liblet_memoryapi:debug_x64 (NMake|liblet_memoryapi|x64_debug|d:\o2\dev\liblet\memoryapi\crt\layermap:PASS-1|d:\o2\dev\otools\inc\otools\makefile.def:6167)}', contains the source file 'd:\o2\Build\oacrdomino\liblet_memoryapi_x64_debug\oacr-merge\summary.log'.
```

In this case PipEA4F8CD3562CB074, declares an output directory of:
` 'd:\o2\Build\oacrdomino\liblet_memoryapi_x64_debug\oacr-merge'`
But at the same time, it or some other pip references a file contained within that directory as though it is a source file:
`'d:\o2\Build\oacrdomino\liblet_memoryapi_x64_debug\oacr-merge\summary.log'`


To solve this, the pip consuming `summary.log` should consume the opaque directory instead of the file.